CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Keep it human-readable, your future self will thank you!
 ### Changed
 
 - Updated configuration examples in documentation and corrected links - [#46](https://github.com/ecmwf/anemoi-training/pull/46)
+- Modified training configuration to support max_steps and tied lr iterations to max_steps by default
```

Comment:
Can you tag the PR #67 please

Reply:
Fixed in commit ef8dbb774abb1038182f6d8bc9791f5fd6d4b477